src/kraken-net/Model/OptimizeRequestBase.cs

```diff
@@ -48,5 +48,8 @@ public SamplingScheme SamplingScheme
 
         [JsonProperty("dev")]
         public bool Dev { get; set; }
+
+        [JsonProperty("quality")]
+        public int Quality { get; set; }
```

Comment:
-Set Quality as Optional parameter. Parameter will be included in JSON message when value will be Provided.
-Added Integration Test method.
-Test old methods without quality parameter.